Mage/Task/AbstractTask.php

```diff
@@ -194,9 +194,12 @@ protected function getParameters()
         }
 
         // if general.yml includes "ssy_needs_tty: true", then add "-t" to the ssh command
-        $needs_tty = ($this->getConfig()->general('ssh_needs_tty', false) ? '-t' : '');
+        $ttyIsNeeded = $this->getConfig()->general('ssh_needs_tty', false)
+                        || $this->getConfig()->deployment('use-sudo', false);
```

Comment:
Use the indentation in the same way as you have used below for string concatenation (with dot `.`)